src/Schema/Directives/Args/OrderByDirective.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+
+namespace Nuwave\Lighthouse\Schema\Directives\Args;
+
+use Nuwave\Lighthouse\Support\Contracts\ArgFilterDirective;
+use Nuwave\Lighthouse\Support\Contracts\ArgDirectiveForArray;
+
+/**
+ * Class OrderBy
+ *
+ * @package App\Http\Graphql\Directives
+ */
+class OrderByDirective implements ArgFilterDirective, ArgDirectiveForArray
+{
+
+    /**
+     * Name of the directive.
+     *
+     * @return string
+     */
+    public function name(): string
+    {
+        return 'orderBy';
+    }
+
+    /**
+     * @param  \Illuminate\Database\Query\Builder|\Illuminate\Database\Eloquent\Builder $builder
+     * @param  string                                                                   $columnName
+     * @param  mixed                                                                    $value
+     *
+     * @return \Illuminate\Database\Query\Builder|\Illuminate\Database\Eloquent\Builder
+     */
+    public function applyFilter($builder, string $columnName, $value)
+    {
+        foreach ($value as $orderBy)
+        {
+            $builder->orderBy($orderBy['field'], $orderBy['order']);
```

Comment:
The directive itself does not require that the type should be `OrderByClause`. 
If that is the case, then it has to check that the `order` is a valid order and also the bigger issue is how does it handle when I input an invalid `field`.

Reply:
Added validation for a correct definition.

Don't see how invalid fields could be handled gracefully?